hydra-node/test/Hydra/Chain/Direct/Contract/Close.hs

```diff
@@ -271,7 +271,8 @@ genCloseMutation (tx, _utxo) =
             [ ChangeValidityInterval (TxValidityLowerBound lowerSlotNo, TxValidityUpperBound upperSlotNo)
             , ChangeOutput 0 $ changeHeadOutputDatum (replaceContestationDeadline adjustedContestationDeadline) headTxOut
             ]
-    , SomeMutation Nothing MutateHeadId <$> do
+    , -- REVIEW: There is no trace for this error. Fails due to `unListData`
```

Comment:
Seems like `unListData` is a builtin. This is definitely unexpected and we should mutate this in a way it does not blow up.

Reply:
Could also be that we just need to access the value differently in the script. Can't say right now.

Reply:
Solved by [commit](https://github.com/input-output-hk/hydra/pull/719/commits/620df63558fe1d769837d0f01de582ce5946c278).